package.json

```diff
@@ -5,13 +5,13 @@
     "main": "lib/index.js",
     "types": "lib/index.d.ts",
     "dependencies": {
-        "long": "4.0.0",
-        "@types/long": "4.0.1"
+        "@types/long": "4.0.1",
+        "long": "4.0.0"
     },
     "devDependencies": {
         "@types/node": "^10.17.28",
-        "@typescript-eslint/eslint-plugin": "^3.10.0",
-        "@typescript-eslint/parser": "^3.10.0",
+        "@typescript-eslint/eslint-plugin": "^4.17.0",
```

Comment:
Since I've updated typescript, the eslint gave a warning about the new typescript version not supported. Thus, I updated eslint plugin too.

Reply:
Can you also verify that the updated dependencies support our minimum supported node version (v10). IIRC, there were some problems before with eslint so we had to use an older version

Reply:
In https://github.com/typescript-eslint/typescript-eslint/blob/v4.17.0/package.json we have 

```
"engines": {
    "node": "^10.12.0 || >=12.0.0"
  }
```

I guess this means node should be at least 10.12 which is a problem for us? Let me investigate other versions of this

Reply:
Then this should be fine. We only use eslint as a devDependency and our PR builder uses Node.js v10.21, so that combination should work just fine